src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -220,6 +225,82 @@ public async Task<LocationSelectViewModel> GetLocationSelectViewModel(LocationSe
             };
         }
 
+        public async Task<byte[]> GetPledgeApplicationsDownloadModel(ApplicationsRequest request)
+        {
+            var result = await _pledgeService.GetApplications(request.AccountId, request.PledgeId);
+
+            var pledgeAppModel = new PledgeApplicationsDownloadModel
+            {
+                Applications = result.Applications?.Select(app =>
+                {
+                    var affordability = GetAffordabilityViewModel(app.Amount, app.PledgeRemainingAmount,
+                        app.NumberOfApprentices, app.MaxFunding, app.StandardDuration, app.StartDate);
+
+                    return new PledgeApplicationDownloadModel
+                    {
+                        DateApplied = app.CreatedOn,
+                        Status = app.Status,
+                        ApplicationId = app.Id,
+                        PledgeId = app.PledgeId,
+                        EmployerAccountName = app.EmployerAccountName,
+                        HasTrainingProvider = app.HasTrainingProvider,
+                        Sectors = app.Sectors ?? new List<string>(),
+                        AboutOpportunity = app.Details,
+                        BusinessWebsite = GetUrlWithPrefix(app.BusinessWebsite),
+                        FormattedEmailAddress = String.Join(";", app.EmailAddresses),
```

Comment:
The ticket appears to ask for a column per email address, not a concatenated field.

Reply:
yeah csv helper doesn't cater for this scenario, so comma separated was the best I could come up with, I'll take another look see if I can meet that aspect of the ticket

Reply:
used the dynamic object to split the locations out